src/services/DinService/DinService.ts

```diff
@@ -0,0 +1,58 @@
+import * as protobuf from 'protobufjs'
+import path from 'path'
+import { RpcChannel } from '../../rpcChannel/RpcChannel'
+import type { serverResponse, serviceRequest } from '../../rpcChannel/ReqRepTypes'
+import { DinPin } from './DinTypes'
+import { StateMsg } from '../serviceTypes/stateMsg'
```

Comment:
can we not use relative paths for import? isn't there a better way to do this : D ?

Reply:
Javascript's module system is different from python's. From what I've found relative paths are used. And it easily auto-completes to relative paths when importing anyway 😆 

ALTHOUGH, it possible to use path aliases by declaring them in typescript config file. But you also need a package to help with that for javascript compilation.